docu/release_notes/2016-12-06-Technev153.html.md

```diff
@@ -0,0 +1,15 @@
+---
+date: December 20, 2016
+service: 'hybris techne'
+official_version: v1.5.3
+headline: 'Avatars and Images Pattern'
+---
+
+### New Pattern
+* New pattern that that displays avatars and images on different context. Markup and iteraction on how to Upload/Change images.
+  ![Tree screenshot](img/v1.5.3/avatars-images.png)
+
+### Minor updates
```

Comment:
We always want to precede bulleted lists with some introduction. In this case, you can write: "This update includes these issue resolutions and feature updates:"